run

```diff
@@ -42,7 +42,7 @@ Commands
 ##
 
 # Define docker images versions
-dev_image="canonicalwebteam/dev:v1.6.1"
+dev_image="canonicalwebteam/dev:v1.6.4"
```

Comment:
Ideally the whole run script should be updated to the latest version [in generator-canonical-webteam](https://github.com/canonical-webteam/generator-canonical-webteam/commits/master/generators/run/templates/run) with:

``` bash
sudo npm install --upgrade yo generator-canonical-webteam
yo canonical-webteam:run
```

(Normally I choose to update `.gitignore` and `run`. but not `package.json`)

Would you be able to do this?